Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -0,0 +1,148 @@
+
+1. Add a [NearInteractionGrabbable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionGrabbable) to that collider.
+
+1. On that object or one of its ancestors, add a component that implements [IMixedRealityPointerHandler](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler). You can for example add [ManipulationHandler](xref:Microsoft.MixedReality.Toolkit.UI.ManipulationHandler).
+
+
+## How to add near touch interactions to collidables
+> If you have UnityUI that you would like to make work with touch events, please read the next section
+
+1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+
+1.  Pick a object that you want to raise pointer events. Any ancestor of that object will be able to receive pointer events. For example, you can add a cube to the scene.
+
+1. Make sure there is a collider to that object.
+
+1. Make sure the layer your object is on is a touchable layer. By default, all layers except Ignore Raycast are touchable. You can see which layers are grabbable by inspecting the Touch Layer Masks in your PokePointer prefab.
+
+1. Add a [NearInteractionTouchable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchable) to your object. 
+
+1. Set "Events to Receive" to Pointer.
+
+1. Click "Fix bounds" and "Fix center"
+
+1. In the scene inspector, you'll see a white outline square and arrow. The arrow pointer to the "front" of the touchable. The collidable will only be touchable from that direction.
+
+1. If you'd like to make your collider touchable from all directions, add a [NearInteractionTouchableVolume](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableVolume) instead.
+
+## How to add near touch interactions to UnityUI
+1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, open your custom profile and go to look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+
+1. Add a UnityUI canvas to your scene.
+
+1. Add a [NearInteractionTouchableUnityUI](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableUnityUI) to your object. 
+
+1. Set "Events to Receive" to Pointer.
+
+
+
+# Examples
```

Comment:
nit: I think this should be ## since the # is the title? And of course then children need to be indented likewise

Reply:
Good catch